packages/components/src/local/form-elements/title-with-icon/index.tsx

```diff
@@ -8,8 +8,11 @@ import PropTypes from './types/props'
 import styles from './styles.module.scss'
 
 /* Render component */
-export const TitleWithIcon: React.FC<PropTypes> = ({ children, forceColor, platformType }) => {
+export const TitleWithIcon: React.FC<PropTypes> = ({ children, forceColor, platformType, icon }) => {
   const iconClassname = getIconClassname(forceColor.toLowerCase(), platformType)
+  console.log('title with icon', forceColor, platformType, icon, iconClassname)
+  // TODO: switch from using `iconClassname` to using the `icon` field in <GetIcon
```

Comment:
Hello @lebaphi - this is where I have got stuck. This component currently uses CSS-classes to specify the icon svg path. But, we have to move to taking that from the data.  I've collated the data here, but can't get the icon to show.

It can be tested from the `storybook` for the component.

Reply:
Hi @IanMayo, I will check it 😉